opennms-services/src/main/java/org/opennms/netmgt/collectd/Collectd.java

```diff
@@ -619,8 +620,6 @@ private void scheduleInterface(OnmsIpInterface iface, String svcName, boolean ex
      *            TODO
      * @param spec
      *            TODO
-     * @param svcName
```

Comment:
I'd opt for not changing the javadoc here.

Reply:
Since the javadoc param TODOs did not help anybody, and just gave warnings in the IDEA I deleted them